RoddyCore/test/src/de/dkfz/roddy/config/validation/DefaultValidatorSpec.groovy

```diff
@@ -0,0 +1,53 @@
+package de.dkfz.roddy.config.validation
+
+import de.dkfz.roddy.config.ConfigurationIssue
+import de.dkfz.roddy.config.ConfigurationValue
+import spock.lang.Specification
+
+import static de.dkfz.roddy.config.ConfigurationIssue.ConfigurationIssueTemplate.*
+
+class DefaultValidatorSpec extends Specification {
+
+    def 'test validation of types'(value, type, expected, expectedErrors) {
+        when:
+        ConfigurationValue cval = new ConfigurationValue('bla', value.toString(), type)
+
+        then:
+        cval.valid == expected
+        cval.warnings == []
+        cval.errors == expectedErrors
+
+        where:
+        value      | type        | expected | expectedErrors
+        1          | 'integer'   | true     | []
+        1.0f       | 'float'     | true     | []
+        1.0        | 'double'    | true     | []
+        true       | 'boolean'   | true     | []
+        'badvalue' | 'boolean'   | true     | [] // Validates always!
+        'a'        | 'integer'   | false    | [new ConfigurationIssue(valueAndTypeMismatch, 'bla', 'integer')]
+        'b'        | 'float'     | false    | [new ConfigurationIssue(valueAndTypeMismatch, 'bla', 'float')]
+        'c'        | 'double'    | false    | [new ConfigurationIssue(valueAndTypeMismatch, 'bla', 'double')]
+        '( abc )'  | 'bashArray' | true     | []
```

Comment:
We should actually consider implementing something for `(abc)` -- without spaces inside the parentheses.

Reply:
Why? What do you mean?

Reply:
Because I ran multiple times into this problem strict requirement when running Roddy.